docs/guide/apm-data-model.asciidoc

```diff
@@ -1,59 +1,33 @@
 [[apm-data-model]]
 == Data Model
 
-Elastic APM agents capture different types of information from within their instrumented applications - namely `transactions`, `spans`, and `errors`. 
+Elastic APM agents capture different types of information from within their instrumented applications - namely `spans`, `transactions`, `errors`, and `metricsets`. 
```

Comment:
Metricsets is not very intuitive. Should we just call it metrics?

Reply:
Good point. I'm looking back at the design document, and it actually says they should be called "application metrics" in documentation, but that may be out of date. Tagging the team that worked on the design doc: @axw @jalvz or @graphaelli. Thoughts on calling it `metrics` vs `metricsets` vs `application metrics` in documentation?

Reply:
+1 on "metrics"; "metricsets" is an implementation detail.

Reply:
Generally agreed, I'm fine either case. We however document implementation details (ie. Intake API), in which it is called `metricsets`.

Reply:
Great, will make the change. Thank you!